tgui/packages/tgui/interfaces/GamePanel.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Stack } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+export function GamePanel(props) {
+  const { act } = useBackend();
+  return (
+    <Window height={240} title="Game Panel" width={280} theme="admin">
+      <Window.Content>
+        <Stack
+          height="100%"
+          vertical
+          align="center"
+          verticalAlign="center"
+          textAlign="center"
+          direction="column"
+          justify="space-around"
+          fillPositionedParent
```

Comment:
```suggestion
          fill
          vertical
```
Слишком много ненужных пропов